### PR TITLE
Maxiaoyu/fix sort

### PR DESCRIPTION
**Pull request by ilovesoup**
Fix sort id. It's not displayed in explain right now causing confusion.

### CODE REVIEW

File: plan/physical_plans.go [hunk 1]
```diff
@@ -222,6 +222,9 @@ func (p *physicalTableSource) tryToAddUnionScan(resultPlan PhysicalPlan) Physica
 	us := &PhysicalUnionScan{
 		Condition: expression.ComposeCNFCondition(p.ctx, append(conditions, p.AccessCondition...)...),
 	}
+	us.tp = Un
```
Comment (hanfei1991) [inline plan/physical_plans.go]: Un is for "Union" but union scan.

Comment (ilovesoup) [inline plan/physical_plans.go]: Changed to UnionScan

**Comment by hanfei1991** [comment]
LGTM

**Comment by shenli** [comment]
LGTM
Please make sure that this pr could pass all the tidb-test.

**Pull request merged**